docs/src/main/docs/grpc/client/01_introduction.adoc

```diff
@@ -0,0 +1,91 @@
+
+1. Creating a `ClientServiceDescriptor` which describes the methods in the service that this client can invoke.
+2. Creating a gRPC `Channel` through which the client communicates with the server.
+
+In later sections in this document, you will see how to customize both `ClientServiceDescriptor` and the `Channel`.
+
+== Quick Start
+
+First, create and run a minimalist `HelloService` gRPC server application as described in the
+https://github.com/aseovic/helidon/blob/master/docs/src/main/docs/grpc/01_introduction.adoc[gRPC Server] documentation.
+
+Assuming that the server is running on port 1408, create a client as follows:
+
+[source,java]
+----
+public static void main(String[] args) throws Exception {
+    ClientServiceDescriptor descriptor = ClientServiceDescriptor.builder(HelloService.class)    // (1)
+                                                                .unary("SayHello")              // (2)
+                                                                .build();
+
+    Channel channel = ManagedChannelBuilder.forAddress("localhost", 1408)                       // (3)
+                                           .usePlaintext()
+                                           .build();
+
+    GrpcServiceClient client = GrpcServiceClient.create(channel, descriptor);                   // (4)
+
+    CompletableFuture<String> future = client.unary("SayHello", "Helidon gRPC!!");                                // (5)
+    System.out.println(future.get());                                                           // (6)
+
+}
+----
+
+1. Create a `ClientServiceDescriptor` for the `HelloService`.
+2. Add the `SayHello` unary method to the `ClientServiceDescriptor`. This method, by default, uses Java serialization for
+marshalling and unmarshalling the request and response values.
+3. Create a gRPC `Channel` that is communicates with the server that is running in localhost and on port 1408 (using plaintext).
+4. Create the `GrpcServiceClient` that uses the above `Channel` and `ClientServiceDescriptor`. `GrpcClientService` represents
+a client that can be used to define the set of methods described by the specified `ClientServiceDescriptor`. In our case, the
+`ClientServiceDescriptor` defines one unary method called `SayHello`.
+5. Invoke the `SayHello` method which returns a `CompletableFuture<String>`.
+6. Print the result.
+
+The example above creates a very simple client to the gRPC server that by default uses Java serialization to marshall
+requests and responses.
+
+We will look into deployment of "standard" gRPC services that use Protobuf for request and response marshalling, as well as
+how you can configure custom marshallers, later in this document.
+
+== Maven Coordinates
+
+The <<getting-started/03_managing-dependencies.adoc, Getting Started>> page describes how you
+should declare dependency management for Helidon applications. Then declare the following dependency in your project:
+
+[source,xml,subs="verbatim,attributes"]
```

Comment:
I'm guessing this was copy pasted from existing docs.
Note that the subs are not needed (we should clean this up to avoid it being copy/pasted).

Reply:
Removed the subs and also removed it in the main gRPC introduction doc.